src/cpp/session/modules/SessionRmdNotebook.R

```diff
@@ -557,6 +557,9 @@ assign(".rs.notebookVersion", envir = .rs.toolsEnv(), "1.0")
    if (length(csvData) == 0 || nrow(csvData) == 0)
       return(list())
    
+   # remove ANSI escapes used for color from text
+   csvData$text <- gsub("\033\\[\\d*(?:;\\d*)?m", "", csvData$text)
```

Comment:
Per my comment "valid ANSI escape to include multiple numbers (2 or more)" the final "?" should be a "*". The "bold" example was just that, there are multiple attributes that can be specified in a single one of these sequences. "Bold italic foreground background" and so forth.

Reply:
Thanks! Sorry I missed that in your original comment.